src/mcare_backend/authapp/tests.py

```diff
@@ -0,0 +1,111 @@
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+from authapp.models import CustomUser, PatientProfile, ExpertProfile
+
+
+data_pass = {
+    'password': 'somepassword'
+}
+
+
+# Create your tests here.
+class CustomUserTestCase(TestCase):
+    def setUp(self):
+        CustomUser.objects.create(
+            email="test@gmail.com",
+            username="ezekieltech",
+            firstname='Ezekiel',
+            lastname='Obha',
+            password=data_password.password
+            role='Patient'
```

Comment:
![age](https://user-images.githubusercontent.com/37231237/82670157-87468680-9c34-11ea-9684-5e498641acb6.png)
On test run using `python manage.py test` , the above error is generated. Kindly give a check on this

Reply:
a comma was omitted just after the the line`password=datapassword.password`.

Reply:
Login requires email and password only.

The default username requirement was changed to use email

Reply:
> ![age](https://user-images.githubusercontent.com/37231237/82670157-87468680-9c34-11ea-9684-5e498641acb6.png)
> On test run using `python manage.py test` , the above error is generated. Kindly give a check on this



> ![age](https://user-images.githubusercontent.com/37231237/82670157-87468680-9c34-11ea-9684-5e498641acb6.png)
> On test run using `python manage.py test` , the above error is generated. Kindly give a check on this

Again, nice catch! Thank you.